ruma-federation-api/src/membership/create_invite.rs

```diff
@@ -0,0 +1,83 @@
+//! Endpoint for inviting a remote user to a room
+
+use js_int::UInt;
+use ruma_events::{room::member::MemberEventContent, EventType};
+use ruma_identifiers::{ServerName, UserId};
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
```

Comment:
Please import-rename for consistency as noted in `CONTRIBUTING.md`.

```suggestion
use serde_json::Value as JsonValue;
```